Libraries/LibJS/Runtime/ArrayPrototype.cpp

```diff
@@ -100,7 +100,8 @@ Value ArrayPrototype::to_string(Interpreter& interpreter)
     for (size_t i = 0; i < array->elements().size(); ++i) {
         if (i != 0)
             builder.append(',');
-        builder.append(array->elements()[i].to_string());
+        if(!array->elements()[i].is_empty())
```

Comment:
Missing a space after `if` here :^)

Reply:
I'm so used to pre-commik hooks doing all the formatting/linting work for me... :smile: